### PR TITLE
LPS-193568 Fix upgrade 7.3 to 7.4

### PR DESCRIPTION
**Pull request by gustavolimav**
https://liferay.atlassian.net/browse/LPS-193568

https://liferay.atlassian.net/wiki/spaces/ENGCOREUPGRADE/pages/1654719711/Upgrade+Development+Training#Choose-the-proper-upgrade-step

### CODE REVIEW

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by joshchong** [comment]
ci:test:sf

**Comment by joshchong** [comment]
ci:test:relevant

**Comment by joshchong** [comment]
ci:test:search

File: modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/upgrade/registry/SXPServiceUpgradeStepRegistrator.java [hunk 1]
```diff
@@ -51,21 +51,46 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
...
 
 		registry.register(
 			"1.3.3", "2.0.0",
 			new com.liferay.search.experiences.internal.upgrade.v2_0_0.
+				DummyUpgradeProcess());
+
+		registry.register(
+			"1.3.0", "2.0.0",
+			new com.liferay.search.experiences.internal.upgrade.v2_0_0.
+				DummyUpgradeProcess());
```
Comment (felipelorenz) [inline modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/upgrade/registry/SXPServiceUpgradeStepRegistrator.java]: @gustavolimav Maybe this is not necessary

**Review by felipelorenz — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/upgrade/registry/SXPServiceUpgradeStepRegistrator.java (hunk 1 above)
Comment (gustavolimav) [reply modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/upgrade/registry/SXPServiceUpgradeStepRegistrator.java]: Pushed already

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:relevant

**Review by felipelorenz — APPROVED** [review]

**Comment by gustavolimav** [comment]
Pull Request forwarded: https://github.com/brianchandotcom/liferay-portal/pull/139671

**Pull request closed without merging**